OpenRA.Game/Settings.cs

```diff
@@ -278,7 +280,6 @@ public class GameSettings
 		public int SelectionDeadzone = 24;
 		public int MouseScrollDeadzone = 8;
 
-		public bool UseClassicMouseStyle = false;
```

Comment:
Could we check for this value in the introduction prompt, and if it is set and `MouseControl` isn't, we'd use this value as the default in the prompt?

Reply:
Can we check for this value in the introduction prompt? Without having it in GameSettings I'm only seeing getting into raw YAML parsing as the solution to this, and I'm not sure we have the settings.yaml handy in this scope

Reply:
A better approach is to add a custom `[LoadUsing()]` to the `MouseControl` field that parses `UseClassicMouseStyle` if it exists.